jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codemodernizer/CodeWhispererCodeModernizerSessionTest.kt

```diff
@@ -39,9 +58,25 @@ class CodeWhispererCodeModernizerSessionTest : CodeWhispererCodeModernizerTestBa
         path.forEach { projectRule.fixture.addFileToModule(module, it, it) }
     }
 
+    @Rule
+    @JvmField
+    val wireMock = WireMockRule(WireMockConfiguration.wireMockConfig().dynamicPort())
+
+    lateinit var gumbyUploadUrlResponse: CreateUploadUrlResponse
+
     @Before
     override fun setup() {
         super.setup()
+        val s3endpoint = "http://127.0.0.1:${wireMock.port()}"
+        gumbyUploadUrlResponse = CreateUploadUrlResponse.builder()
+            .uploadUrl(s3endpoint)
+            .uploadId("1234")
```

Comment:
Any benefit to makes these variables?

Reply:
These are to mock the http request for uploading artifact to s3. I can move these to the function `test uploadPayload()` where they are actually been used if needed.